src/ComponentConfig.js

```diff
@@ -54,17 +54,79 @@ module.exports = class ComponentConfig {
 
         return Object.entries(services).reduce((newConfig, [serviceName, config]) => {
             const normalizedConfig = this.normalizeService(serviceName, config);
+            // eslint-disable-next-line no-param-reassign
             newConfig[serviceName] = normalizedConfig;
             return newConfig;
         }, {});
     }
 
+    static modelIsExposed(name, modelsConfig) {
+        // if there is no configuration all models are exposed
+        if (!modelsConfig) {
+            return true;
+        }
+        const config = modelsConfig[name];
+        // false or not set
+        if (!config) {
+            return false;
+        }
+        if (config === true) {
+            return true;
+        }
+        return config.expose !== false;
+    }
+
+    static modelIsPublic(name, modelsConfig) {
+        // all models are exposed by default
+        if (!modelsConfig) {
+            return false;
+        }
+        const config = modelsConfig[name];
+        // false, not set or set to true (legacy configuration)
+        if (!config || config === true) {
+            return false;
+        }
+        return config.isPublic === true;
+    }
+
+    static modelIsGlobal(name, modelsConfig) {
+        // default behavior
+        if (!modelsConfig) {
+            return true;
+        }
+        const config = modelsConfig[name];
+        // false or not set
+        if (!config) {
+            return false;
+        }
+        // default behavior if set to true
+        if (config === true) {
+            return true;
+        }
+        return config.isGlobal !== false;
```

Comment:
Same here as with the exposed configuration

Reply:
Same here, `isGlobal` is true by default (I tried to preserve backwards compatibility in this case). Otherwise the translations which have a hardwired dependency on the Locale model will fail and we'd have to reconfigure all our services.